docs/tasks.md

```diff
@@ -160,7 +160,8 @@ To update a task assignment, call the
 `TasksManager.UpdateTaskAssignmentAsync(BoxTaskAssignmentUpdateRequest taskAssignmentUpdateRequest)`
 method.  This can be used to resolve or complete a task.
 
-<!-- sample put_task_assignments_id -->
+Updating the resolution state:
+<!-- sample put_task_assignments_id resolution_state-->
```

Comment:
@PreciselyAlyss and dito here. We need at least one "default" sample per endpoint